### PR TITLE
Feat/refactor stores

### PR DESCRIPTION
**Pull request by andredaa**

### CODE REVIEW

**Comment by totev** [comment]
I think the component and it's store are in a good-enough stage to be merged with the master branch. This feature branch has lived long enough :)

File: src/components/Scenario/ViewConfiguration.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export interface ViewConfiguration {
   filter: boolean;
 }
+// TODO DOBO, wat macht das??
```
Comment (andredaa) [inline src/components/Scenario/ViewConfiguration.ts:4]: @totev was macht das? :P

Comment (andredaa) [reply src/components/Scenario/ViewConfiguration.ts:4]: ist das noch relevant, nachdem wir das "filter" feature gelöscht haben?

**Review by andredaa — COMMENTED** [review]
Let's just check if view still need that viewInterface.

**Comment by andredaa** [comment]
it is already quite a big diff.
i would suggest merging this. then merge the compatible "rainFromBackend" afterwards.
Actually this contains the "delete features" branch - which we could merge before this one.

File: src/components/Scenario/ViewConfiguration.ts (hunk 1 above)
Comment (totev) [reply src/components/Scenario/ViewConfiguration.ts:4]: if the filter isn't there, then I guess this is obsolete as well :)

Comment (andredaa) [reply src/components/Scenario/ViewConfiguration.ts:4]: I didnt find any usages. i deleted it :)

**Pull request merged**